sympy/sets/tests/test_sets.py

```diff
@@ -1046,6 +1046,16 @@ def test_issue_10931():
     assert S.Integers - S.Reals == EmptySet()
 
 
+def test_issue_11442():
+    a = Symbol('a', real=True)
+    b = Symbol('b', real=True)
+    i1 = Interval(2, 7)
+    i2 = Interval(a, b)
+    assert not i1.is_EmptySet
```

Comment:
I would prefer `False`(Python bool), since the answer as a thumb rule whenever the answer could be three possible values (None, False, True), use the Python bool, otherwise go with SymPy bool.

Also I think it is better to use `==` rather than `is` may be, (since Python bool should preferably be compared with `==` rather than `is`)